README.md

```diff
@@ -41,6 +41,7 @@ You need the following permissions to run this module.
 - [ Complete example with byok encryption, CBR rules and storing credentials in secrets manager](examples/complete)
```

Comment:
I think we should add a section on read-only replica scenario in the main README. Highlighting the fact that there are additional considerations when setting the variables in the module and pointing to https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-read-only-replicas . Best if we could even have a table with the restrictions listed at that page https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-read-only-replicas and the name of the variables in the module.

Going further (probably not in this PR) it may be good to try to default or validate further the inputs of the module so that we can fail execution early (at terraform plan time) whenever possible. Eg: it should be possible to fetch information about the leader instance member config and use this as a default input rather than requesting to pass this as an input. Alternatively, at least validate that the member specs passed as input for readonly replica are >= that the leader ones.

Reply:
Added some of the applicable considerations and also pointed to the IBM cloud documentation link.